### PR TITLE
Add new estimation method and enforce order of return in the collect functions in the staking contracts

### PR DESCRIPTION
**Pull request by vtleonardo**
This PR:
- Add a new function to estimate the profits and ETH and ALCA in 1 call. 
- Enforce that the eth profit is returned first and the token after in the collect and estimation functions. We are adopting the same order that we use when we burn a position for consistency.

### CODE REVIEW

File: bridge/contracts/libraries/StakingNFT/StakingNFT.sol [hunk 1]
```diff
@@ -297,6 +297,19 @@ abstract contract StakingNFT is
         return payout;
     }
 
+    /// estimateAllProfits returns the amount of AToken a tokenID may withdraw
+    function estimateAllProfits(uint256 tokenID_)
```
Comment (z-j-lin) [inline bridge/contracts/libraries/StakingNFT/StakingNFT.sol:301]: add unit test

**Review by z-j-lin — CHANGES_REQUESTED** [review]
add unit tests for new functions

File: bridge/contracts/libraries/StakingNFT/StakingNFT.sol (hunk 1 above)
Comment (vtleonardo) [reply bridge/contracts/libraries/StakingNFT/StakingNFT.sol:301]: Unit tests added

**Review by Cr0wn-Gh0ul — COMMENTED** [review]
Change _collect function name to be more correct.

**Review by Cr0wn-Gh0ul — APPROVED** [review]
LGTM

**Pull request merged**